types/node/index.d.ts

```diff
@@ -568,6 +568,8 @@ declare namespace NodeJS {
         setegid(id: number | string): void;
         getgroups(): number[];
         setgroups(groups: Array<string | number>): void;
+        setUncaughtExceptionCaptureCallback((err: Error) => void | null): void
```

Comment:
The name of the parameter is missing.
Additionally I think that this specifies a function which returns `void  | null` and not a Function `(err: Error) => void` or `null`.